blackbox/docs/administration/ssl.txt

```diff
@@ -114,3 +114,272 @@ string. E.g.::
 For further information visit `psql documentation`_.
 
 .. _psql documentation: https://www.postgresql.org/docs/9.2/static/app-psql.html
+
+
+Setting up a Keystore/Truststore with a certificate chain
+=========================================================
+
+In case you need to setup a Keystore/Trustore, here are the commands
+to get you started. All the commands use a validity of 36500 days
+(about 100 years). You might want to use less.
+
+.. NOTE::
+
+   Only follow the optional steps if you want to create your own
+   Certificate Authority. Otherwise, please request a signed certificate
+   from one of the CAs bundled with Java.
+
+
+Generate Keystore with a Private Key
+------------------------------------
+
+The first step is to create a keystore with a private key.
+
+Command::
+
+    keytool -keystore keystore -genkey -alias server -validity 36500
+
+Output::
+
+    Enter keystore password:
+    Re-enter new password:
+    What is your first and last name?
+      [Unknown]:  CrateDB
+    What is the name of your organizational unit?
+      [Unknown]:  PSQL Netty SSL
+    What is the name of your organization?
+      [Unknown]:  crate.io
+    What is the name of your City or Locality?
+      [Unknown]:  Dornbirn
+    What is the name of your State or Province?
+      [Unknown]:  Vorarlberg
+    What is the two-letter country code for this unit?
+      [Unknown]:  AT
+    Is CN=CrateDB, OU=PSQL Netty SSL, O=crate.io, L=Dornbirn, ST=Vorarlberg, C=AT correct?
+      [no]:  yes
+
+    Enter key password for <server>
+        (RETURN if same as keystore password):
+    Re-enter new password:
+
+
```

Comment:
I find it easier to read the source with two lines between the sections. It doesn't affect the output. If you don't mind, I'll keep it.